schema_based_on_diagram.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE patients (
+    id int PRIMARY KEY,
+    name TEXT NOT NULL,
+    date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE invoices (
+    id int PRIMARY KEY,
+    total_amount DECIMAL(5, 2) NOT NULL,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id int
+);
+
+CREATE TABLE medical_histories (
+    id int PRIMARY KEY,
+    admitted_at timestamp,
+    patient_id int,
+    status VARCHAR(100)
+);
+
+CREATE TABLE invoice_items (
+    id int PRIMARY KEY,
+    unit_price DECIMAL(5,2) NOT NULL,
+    quantity int,
+    total_price DECIMAL(5,2) NOT NULL,
+    invoice_id int,
+    treatment_id int
+);
+
+CREATE TABLE treatments (
+    id int PRIMARY KEY,
+    type VARCHAR(100),
+    name VARCHAR(100)
+);
+
+CREATE TABLE med_history_treatment (
+    med_history_id INT,
+    treatment_id INT    
+);
+
+ALTER TABLE medical_histories
+ADD CONSTRAINT patient_fk FOREIGN KEY(patient_id)
+REFERENCES patients(id);
+
+ALTER TABLE invoices
+ADD CONSTRAINT medical_history_id_fk FOREIGN KEY(medical_history_id)
+REFERENCES medical_histories(id);
+
+ALTER TABLE invoice_items
+ADD CONSTRAINT invoice_fk FOREIGN KEY(invoice_id)
+REFERENCES invoices(id);
+
+ALTER TABLE invoice_items
+ADD CONSTRAINT treatment_fk FOREIGN KEY(treatment_id)
+REFERENCES treatments(id);
+
+ALTER TABLE med_history_treatment
+ADD CONSTRAINT med_history_id_fk FOREIGN KEY(med_history_id)
+REFERENCES medical_histories(id);
+
+ALTER TABLE med_history_treatment
+ADD CONSTRAINT treatment_id_fk FOREIGN KEY(treatment_id)
+REFERENCES treatments(id);
+
+CREATE INDEX index_treatment ON invoice_items(treatment_id);
+
+CREATE INDEX index_medical_history ON invoices(medical_history_id);
+
+CREATE INDEX index_invoice ON invoice_items(treatment_id);
+
+CREATE INDEX index_patient ON medical_histories(patient_id);
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] You did great creating indexes for the `FOREIGN KEYS` in each table , but you did create index for the Keys in the `med_history_treatment` joint table. I'll suggest you add the create the indes for the two ids. :+1: :waffle: